app/services/changes/tracker.rb

```diff
@@ -0,0 +1,58 @@
+# class ChangeTrackerService
```

Comment:
why not remove the comments?

Reply:
My bad! I came across this article (https://www.nopio.com/blog/how-to-create-an-api-wrapper-of-an-external-service-in-rails/) and tried to apply some ideas to the project and i duplicated the code to manually test it and then i forgot to erase.